src/components/PetList.js

```diff
@@ -242,16 +259,56 @@ export default function PetList(props) {
               />
             </div>
           </div>
+          <div className='row mb-2'>
+            <div className='col-sm m-1'>
+              {page == 'pets' ?
+                <Select
+                  placeholder='DISTANCE'
+                  name='distance'
+                  id='distance'
+                  value={distance}
+                  options={distanceOptions}
+                  onChange={selectedOption => {
+                    setConfirmSaved(false);
+                    inputChangeHandler('distance', selectedOption)
+                  }}
+                /> : ''}
+            </div>
+            <div className='col-sm m-1 d-flex align-items-start'>
+              {page == 'pets' ?
+                <>
+                  <label className='mt-2' htmlFor='zipCode'>near </label>
```

Comment:
I think the "near" here looks a bit clunky.
![image](https://user-images.githubusercontent.com/46770903/156234452-5030fdea-c865-4f85-999c-588f6f384a10.png)

Do you think it's clear enough with out the "near", and just having the distance preference and the zip code? Additionally, we could change the distance label from "10 miles" to "10 miles from" or something